clusters/build-clusters/01_cluster/openshift-image-registry/README.md

```diff
@@ -0,0 +1,19 @@
+# openshift-image-registry
+
+## Configuration
+
+Doc: [expose the registry using DefaultRoute](https://docs.openshift.com/container-platform/4.2/registry/securing-exposing-registry.html#registry-exposing-secure-registry-manually_securing-exposing-registry)
```

Comment:
Hmm what do we need the expose for? At least for now we'll only need in-cluster access to the images, no?

Reply:
That is correct. We do not have to.
It is convenient to expose it when testing the image. I can pull it with docker/podman.
Do not think it going to harm us badly. On 3.x ... it is exposed by default.